reactioncommerce/reaction-next-starterkit
# (feat): New Profile Orders page

Expands upon #499 to add Orders data to profile page
Impact: **minor**
Type: **feature**

## Issue
Add Orders data to Account Profile

## Solution
- Split existing orders page into `profileAddressBook` and `profileOrders` pages. This is necessary due to the way we do pagination, our existing method broke when using `asPath` to display different components (`Orders` or `AddressBook`) on a single page based on the URL.

## Breaking changes
None

## Testing

- Visit `/profile/orders` as a non-logged in user, and see a 404 / page not found
- Login as a user, create 6+ orders (to see pagination)
- Visit `/profile/orders` as a logged in user, see that orders show up and match our designs: https://zpl.io/V0ypYWR
- Move orders into different workflow statuses
- See orders are filtered as they should
